README.adoc

```diff
@@ -273,14 +279,16 @@ Tests run: 2, Failures: 0, Errors: 0, Skipped: 0
 
 To see whether the tests detect a failure, change the [hotspot=endpoint file=0]`endpoint` for the `inventory` service in
 the [hotspot file=0]`src/main/java/io/openliberty/guides/inventory/InventoryResource.java` file to something else, then
-run the Maven build again. You see a test failure occur.
+run the tests again to see a test failure occur.
```

Comment:
Change
"To see whether the tests detect a failure, change the [hotspot=endpoint file=0]`endpoint` for the `inventory` service in the [hotspot file=0]`src/main/java/io/openliberty/guides/inventory/InventoryResource.java` file to something else, then run the tests again to see a test failure occur."
To
"To see whether the tests detect a failure, change the [hotspot=endpoint file=0]`endpoint` for the `inventory` service in the [hotspot file=0]`src/main/java/io/openliberty/guides/inventory/InventoryResource.java` file to something else. Then, run the tests again to see that a test failure occurs."